apps/web/src/hooks/useMerkl.tsx

```diff
@@ -29,9 +29,30 @@ export function useMerklInfo(poolAddress: string | null): {
   } | null
   hasMerkl: boolean
   refreshData: () => void
+  merklApr?: number
 } {
   const { account, chainId } = useAccountActiveChain()
 
+  const { data: merklData } = useQuery({
+    queryKey: ['merklAprData', chainId, poolAddress],
+    queryFn: async () => {
+      try {
+        const resp = await (
+          await fetch(`https://api.angle.money/v2/merkl?chainIds[]=${chainId}&AMMs[]=pancakeswapv3`)
+        ).json()
+        return resp
```

Comment:
the best practice of `fetch` is, always check `response.ok` is `true`

```ts

const response = await fetch()

if (response.ok) return response.json()

throw response

```

Reply:
Ok, let me adjust it